components/transcription/TranscriptionDialog.vue

```diff
@@ -0,0 +1,139 @@
+<script setup lang="ts">
+import type { TrackModel } from "@bcc-code/bmm-sdk-fetch";
+import getSongtreasuresLink from "~/utils/songtreasures";
+
+const props = defineProps<{
+  track: TrackModel;
+}>();
+
+const show = defineModel<boolean>("show", { default: false });
+
+const { data: transcription, execute } = useTrackTranscription({
+  id: props.track.id,
+});
+
+const { $appInsights } = useNuxtApp();
+
+watch(
+  show,
+  (_show) => {
+    if (!_show) return;
+
+    execute();
+    $appInsights.event("Transcription dialog opened", {
+      trackId: props.track.id,
+      trackSubtype: props.track.subtype,
+    });
+  },
+  { once: true },
+);
+
+const lyricists = computed(() =>
+  props.track.contributors
+    ?.filter((c) => c.type === "lyricist")
+    .map((c) => c.name)
+    .join(", "),
+);
+
+const composers = computed(() =>
+  props.track.contributors
+    ?.filter((c) => c.type === "composer")
+    .map((c) => c.name)
+    .join(", "),
+);
+
+const { t } = useI18n();
+
+const dialogTitle = computed(() => {
+  if (!trackIsSong(props.track))
+    return t("transcription.auto-transcribed.title");
+  return t("transcription.lyrics");
```

Comment:
Since you're already changing them anyway. Can you change the order of the buttons:
- Play next
- Add to queue
- Transcription / Lyrics
- More information
- Add to playlist
- Go to album
- Go to contributors
- Share track
- Download